roles/cockpit-ui/tasks/main.yml

```diff
@@ -41,37 +41,45 @@
   run_once: true
   when: check_docker_registry_exists.rc == 0
 
-- name: Retrieve docker-registry route
+- name: Retrieve docker-registry json
   oc_route:
     name: docker-registry
     namespace: default
     kubeconfig: "{{ openshift_hosted_kubeconfig }}"
```

Comment:
With https://github.com/openshift/openshift-ansible/pull/3290 being merged, you should be able to just point to the admin kubeconfig here and avoid the mktemp/delete tasks.